src/main/kotlin/com/hedvig/underwriter/service/UnderwriterImpl.kt

```
@@ -141,8 +141,9 @@ class UnderwriterImpl(
                 ).price
             }
             is DanishAccidentData -> {
-                // TODO: fix when pricing is in place
-                Money.of(6666, "DKK")
+                priceEngineService.queryDanishAccidentPrice(
+                    PriceQueryRequest.DanishAccident.from(quote.id, quote.memberId, quote.data)
+                ).price
             }
             is DanishTravelData -> {
```

Maybe we should enable DanishTravel now as well, since you also added support for that in price-engine. While not provisioned in price-engine, any request with it should return an error, so should be "safe".